README.md

```diff
@@ -8,3 +8,9 @@ Don't hesitates to add yours via PR.
 | Emoji | Function/Value |
 | --- | --- |
 | c╯°□°ↄ╯ | `throw` |
+|  ✅, 👍, 👌 | `true` |
+|  👎 | `false` |
+| 😻, ⬇️😻, ➡️😻, 😹, ⬇️😹, ➡️😹, 🐈, ⬇️🐈, ➡️🐈 | `cat` |
```

Comment:
Is that correct? You used the same emojis for `cat` and `vcat/hcat`

Reply:
Nope, fixed it!